CODEOWNERS

```diff
@@ -3,6 +3,7 @@
 
 /consul*/ @grafana/mimir-maintainers @grafana/loki-team
 /etcd-operator/ @grafana/mimir-maintainers
+/jaeger-agent-mixin/ @grafana/grafana-alloy-maintainers
```

Comment:
GitHub says this is an "unknown owner"... I'm not sure why this is and how to fix it? Is there an actual issue, or is GitHub just confused?

Reply:
Added the team to the repo so they have write access :)